src/codec_avm.c

```diff
@@ -536,8 +536,12 @@ static avifResult avmCodecEncodeImage(avifCodec * codec,
                                       int quantizer,
                                       avifEncoderChanges encoderChanges,
                                       avifAddImageFlags addImageFlags,
+                                      avifBool disableLaggedOutput,
                                       avifCodecEncodeOutput * output)
 {
+    // TODO(vigneshv): add support for avm lagged output when necessary. Ignore it for now.
+    (void)disableLaggedOutput;
```

Comment:
We can use the same code as codec_aom.c.

Reply:
we probably can, but i don't know if libavm respects this flag and produces the intended output as of now. so i just left it as a TODO for later so that it can be verified before it is enabled.

in general, i think of codec_avm as a fork of codec_aom at that point in time. i don't think we need to keep codec_aom and codec_avm arbitrarily in sync without knowing if things actually work.

Reply:
I understand your point. But it will be less work to just port the codec_aom.c change now.

Note that this file already sets `cfg->g_lag_in_frames` to 0 at line 631 and line 641. So if `cfg->g_lag_in_frames == 0` doesn't work, you won't be making it worse.

Reply:
> Note that this file already sets `cfg->g_lag_in_frames` to 0 at line 631 and line 641.

Yes, because it was forked from codec_aom that way. :)

> So if `cfg->g_lag_in_frames == 0` doesn't work, you won't be making it worse.

Anyways, done.